systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -248,6 +250,16 @@ private void testDockerImagesForKafkaConnect() {
         LOGGER.info("Docker images verified");
     }
 
+    @BeforeEach
+    void createTestResources() {
+        createResources();
+    }
+
+    @AfterEach
+    void deleteTestResources() throws Exception {
+        deleteResources();
```

Comment:
Why does this one not need ` waitForDeletion(TEARDOWN_GLOBAL_WAIT, NAMESPACE);`?

Reply:
Because there is a kafka cluster which is active during execution of whole class. I have an idea that `waitForDeletion(TEARDOWN_GLOBAL_WAIT, NAMESPACE)` can accept pod name or prefix and it will check deletion of specific pods - kafka-connect in that case.

Reply:
Shouldn't you have a `waitForDeletion(TEARDOWN_GLOBAL_WAIT, NAMESPACE);` in some `@AfterAll` method in that case?

Reply:
It doesn't matter, after all tests whole namespace will be deleted so it does not affect other tests. We can add `waitForDeletion` there just to make sure that all pods are being removed correctly after tests, what do you think?